paasta_tools/cli/cmds/metastatus.py

```diff
@@ -64,6 +64,13 @@ def add_subparser(subparsers):
         help="define a different soa config directory",
     )
     status_parser.add_argument(
+        '-a', '--autoscaling-info',
```

Comment:
Yep, it's a little slow and it breaks if the credentials aren't found/are invalid. But we could catch that exception really. Let's see how it goes and maybe make it default later.